### PR TITLE
add a strict note about the compilers run-time

### PR DESCRIPTION
**Pull request by ocefpaf**
Ping @xylar

### CODE REVIEW

File: src/user/tipsandtricks.rst [hunk 1]
```diff
@@ -61,6 +61,15 @@ Here is how a ``.condarc`` file would look like:
 In addition to the channel priority we recommend to always install your packages inside a new environment instead the root environment from anaconda/miniconda.
 Using envs make it easier to debug problems with packages and ensure the stability of your root env.
 
+Note: in the past ``conda-forge`` used to vendorize some of ``defaults`` dependencies that were not built in our infrastructure,
```
Comment (croth1) [inline src/user/tipsandtricks.rst]: Can we put this in a `..note::` box for consistency?

File: src/user/tipsandtricks.rst [hunk 2]
```diff
@@ -61,6 +61,15 @@ Here is how a ``.condarc`` file would look like:
...
+like compilers run-times, to avoid the mixing channel problem.
+However, with the ``strict`` option, we no longer have to vendorize those (which has its own problems too),
```
Comment (croth1) [inline src/user/tipsandtricks.rst]: Do we now recommend everybody to use strict? If so we probably should also put that here: http://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge

Comment (croth1) [inline src/user/tipsandtricks.rst]: I personally would leave out `(which has its own problems too)` but I guess that's a matter of taste.

Comment (xylar) [reply src/user/tipsandtricks.rst]: Yes, I agree.

Comment (xylar) [reply src/user/tipsandtricks.rst]:
```suggestion
However, with the ``strict`` option, we no longer have to vendorize those (this led to its own set of problems),
```
@croth1 it seems like a good idea to indicate why these channels are marked as `broken` and effectively not available.  My preference would be for some brief explanation of what those problems where but it seems good to at least indicate that there was a good reason not to keep them available, given that it's caused some inconvenience for people like me that where told to use only the `conda-forge` channel before the `strict` option was available.

Comment (ocefpaf) [reply src/user/tipsandtricks.rst]: In `conda 4.7` where `strict` will be default and we will need to update that but I guess that, b/c mixing channels without `strict` is broken, we should add it there too.

File: src/user/tipsandtricks.rst (hunk 1 above)
Comment (ocefpaf) [reply src/user/tipsandtricks.rst]: Done. Thanks!

File: src/user/tipsandtricks.rst (hunk 2 above)
Comment (ocefpaf) [reply src/user/tipsandtricks.rst]: Done! Not sure if we should leave the comment about conda 4.7 there though.

Comment (ocefpaf) [reply src/user/tipsandtricks.rst]: Maybe a better explanation for this should be in the `News` section?

**Comment by ocefpaf** [comment]
@croth1 and @xylar this is ready for a second round of review/merge.

File: src/user/tipsandtricks.rst (hunk 2 above)
Comment (xylar) [reply src/user/tipsandtricks.rst]: Sure, that's fine.

**Comment by xylar** [comment]
Yep, looks good to me.

**Comment by croth1** [comment]
Thanks @ocefpaf @xylar for working on this!

**Pull request merged**